src/main/cpp/Robot.cpp

```diff
@@ -82,6 +99,32 @@ void Robot::ExpectAutonomousEndConds() {
     }
 }
 
+void Robot::ClimbingSequenceSM() {
+    switch (m_state) {
+        case ClimbingStates::kSecondRung: {
+            solenoidTimer.Start();
+            if (solenoidTimer.HasElapsed(1_s)) {
+                climber.TelescopingOut();
+            } else if ((!barSensor.Get()) && (!upperSensor.Get())) {
+                climber.TelescopingExtention(0.65);
+            }
+
+            if ((barSensor.Get()) && (!lowerSensor.Get())) {
+                climber.TelescopingExtention(-0.65);
+            }
+
+            if (lowerSensor.Get()) {
```

Comment:
Could consider turning these into individual getter methods in the Climber class